src/common/RunParams.cpp

```diff
@@ -718,18 +729,18 @@ void RunParams::parseCommandLineOptions(int argc, char** argv)
 void RunParams::printHelpMessage(std::ostream& str) const
 {
   str << "\nUsage: ./raja-perf.exe [options]\n";
+
   str << "Valid options are:\n";
 
-  str << "\t --help, -h (print options with descriptions)\n\n";
+  str << "\t --help, -h (print available command line options with descriptions)\n\n";
 
-  str << "\t --show-progress, -sp (print execution progress during run)\n\n";
+  str << "\t Print info about option arguments....\n"
```

Comment:
I didn't understand what this meant the first few times I read it. Should it say something more like "Options to prints info ..."

Reply:
Good point. I changed it.

Reply:
Can you add something like "Options to ..." to the other category intros?

Reply:
Done. If you're OK with this, I will move it out of draft stage so all CI checks will run